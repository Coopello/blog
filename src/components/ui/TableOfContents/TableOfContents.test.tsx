import renderer from "react-test-renderer";

import { TableOfContents } from "./";

describe("ui/TableOfContents", () => {
  it("Snap Shot", () => {
    const component = renderer.create(
      <TableOfContents contents={["hoge", "piyo", "fuga"]} />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
