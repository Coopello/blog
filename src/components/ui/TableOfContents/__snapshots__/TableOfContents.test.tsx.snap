// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/TableOfContents Snap Shot 1`] = `
<nav>
  <ul
    className="flex flex-col gap-2"
  >
    <li
      className="h-full w-full"
    >
      <a
        className="box-border block h-full w-full px-4 text-xl hover:bg-black/5"
        data-val="toc"
        href="#hoge"
      >
        hoge
      </a>
    </li>
    <li
      className="h-full w-full"
    >
      <a
        className="box-border block h-full w-full px-4 text-xl hover:bg-black/5"
        data-val="toc"
        href="#piyo"
      >
        piyo
      </a>
    </li>
    <li
      className="h-full w-full"
    >
      <a
        className="box-border block h-full w-full px-4 text-xl hover:bg-black/5"
        data-val="toc"
        href="#fuga"
      >
        fuga
      </a>
    </li>
  </ul>
</nav>
`;
